depends/packages/native_clang.mk

```diff
@@ -1,12 +1,12 @@
 package=native_clang
-$(package)_version=11.1.0
+$(package)_version=15.0.6
```

Comment:
fa0fcb35f3d6c886dfc7af0e036fb1942650fca0:

> There is no x86_64 binaries for 15.0.7.

https://github.com/llvm/llvm-project/releases/download/llvmorg-15.0.7/clang+llvm-15.0.7-x86_64-apple-darwin21.0.tar.xz ?

Reply:
Those binaries are built for Darwin.

Reply:
Indeed. Sorry for noise.